src/main/java/org/cyclonedx/maven/BaseCycloneDxMojo.java

```diff
@@ -80,7 +80,7 @@ public abstract class BaseCycloneDxMojo extends AbstractMojo {
      *
      * @since 2.1.0
      */
-    @Parameter(property = "schemaVersion", defaultValue = "1.6", required = false)
+    @Parameter(property = "schemaVersion", defaultValue = "1.5", required = false)
```

Comment:
until now, default schema version was latest supported: why doing that? is really consistency with Gradle plugin a good reason?

Reply:
@msymons WDYT?

Reply:
I don't have a strong opinion but thought there might be precedence for keeping the default one version behind. Maybe this would be a good addition to the missing CONTRIBUTING.md document :)

Reply:
I believe we are making CycloneDX 1.6 the default only in v2.9.x, correct?

Reply:
please drop this commit and squash the others

Reply:
I have no objection to updating default to 1.6.  As @hboutemy observes, this maintains consistent behaviour with previous releases.   The 1.6 spec was released 6 months ago.  It will seem like no time at all before 1.7 is out!

Reply:
Agreed that 1.6 should be the default. Its been out long enough and is a higher-quality spec over v1.5.